api/MediaError.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "3.5"
+            "version_added": "4"
```

Comment:
The original "3.5" matches the data for api.HTMLMediaElement.error which is an instance of `MediaError`. This is more like an interface object not exposed. Can you check this manually?

Reply:
I've tested http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8806 in Firefox 3.6 on BrowserStack and `video.error` is there. To make sure it wasn't just always null I checked when it was added and as part of that came across https://bugzilla.mozilla.org/show_bug.cgi?id=589561#c4. `MediaError` was originally `HTMLMediaError` and `TimeRanges` was `HTMLTimeRanges`.

I've sent https://github.com/mdn/browser-compat-data/pull/8734 to fix this.